### ISSUE TITLE
how many epochs needed

### ISSUE THREAD

**Issue by LeeKuo1990** [bbbbbbzhou/DuDoRNet#2]
I have trained your model from the scratch, but at the first 7 epochs (up to now), the images generated are strange, very different from the target images, and i cannot see any tissue structure, is that right?

**Comment by bbbbbbzhou** [comment]
Hi, the model's convergence behavior may depend on the data you are using. If the loss doesn't converge as the epoch increase, please try to adjust the hyper-parameters, such as learning rate. Please also ensure the data preparation is correct, as DuDoRNet also involves FT and FFT transformation that needs to match the input data.
